tasks/main.yml

```diff
@@ -126,15 +101,54 @@
   when: postgresql_pg_hba_conf is defined
   notify: Restart postgresql
 
+- name: Create postgresql directory in /etc
+  file:
+    path: /etc/postgresql
+    state: directory
+    mode: '0755'
+
+- name: Prepare internal file for configuration
+  copy:
+    content: ""
+    dest: "/etc/postgresql/system-roles-internal.conf"
+    force: true
+    mode: 0600
+    owner: postgres
+    group: postgres
+  notify: Restart postgresql
+
+- name: Generate internal postgresql system-roles.conf
+  when: postgresql_server_tuning or postgresql_ssl_enable
+  block:
+    - name: Link generated conf file with server one
+      lineinfile:
+        path: /var/lib/pgsql/data/postgresql.conf
```

Comment:
don't you need a `regexp` to go with `line`?  https://docs.ansible.com/ansible/2.9/modules/lineinfile_module.html#examples

Reply:
Sorry, I didn't catch the idea behind it. I just need to add one static line at the end of the file. It is the correct way? or should I use regexp somehow?

Reply:
ah, ok - so there will never be a line in the file that includes system-roles-internal.conf
It isn't clear to me from reading the documentation what happens if you have `line` with no other specifiers e.g. will it always add at the BOF or EOF?  I guess if the file is empty it doesn't matter.  But consider adding `insertbefore: BOF` if you want to make it explicit that you want this line inserted at the beginning of the file.

Reply:
Ah ok, that makes sense, it seems that it works with EOF by default and it fits my case, but yes I will add it to make it clear.